pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -284,6 +285,7 @@ func (l *listUsersQuery) expandDirect(
 	pool.WithCancelOnError()
 	pool.WithMaxGoroutines(int(l.resolveNodeBreadthLimit))
 
+	var errs *multierror.Error
```

Comment:
In the latest versions of Go, specifically from Go 1.19 onwards, you can use the `errors.Join` function, which was added to the standard library. This function can replace all of the functionality of the 3rd party `multierror` package.

Any chance we can use that for new code and have a transition plan for older code?

The `errors.Join` function in Go allows you to combine multiple errors into a single error. If all errors are nil, `errors.Join` returns nil. If only one error is non-nil, it returns that error. If there are multiple non-nil errors, it returns an error that formats as the concatenation of the individual errors' messages.

Example:

```go
    err1 := errors.New("error one")
    err2 := nil
    err3 := errors.New("error three")

    joinedErr := errors.Join(err1, err2, err3)
    if joinedErr != nil {
        fmt.Println(joinedErr) // Output: error one; error three
    }
```

Reply:
> have a transition plan for older code?

open a ticket and tag it "good first issue" 😄

Reply:
https://github.com/openfga/openfga/issues/1556